Android/app/src/androidTest/java/tie/hackathon/travelguide/Tests/DemoTest.java

```diff
@@ -13,7 +15,18 @@
 public class DemoTest extends EspressoTestBase {
 
     @Test
-    public void testDemo() throws Exception {
-        onView(withContentDescription("SIGNUP")).perform(click());
+    public void testIfSignUpIsSuccessful() throws Exception {
+        LoginPage.signUp("user", "0754211009", "test1234");
+        LoginPage.tapSignUp();
+        MainPage.signOut();
+        Assert.assertTrue("The login page is not displayed", Helpers.isIdDisplayed(R.id.ok_signup));
```

Comment:
"Helpers.isIdDisplayed()", nu-mi spune nimic. nu este sugestiv ce element verifici tu aici.  Aici trebuie sa verifici ceva dintr-un screen (SignIn cred). Metodele din "Helpers" le apelem in metodele in "screens", iar in teste apelam metodele din "screens".